specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json

```diff
@@ -7535,11 +7544,27 @@
         "modelAsString": true
       }
     },
+    "DiffDiskPlacement": {
+      "type": "string",
+      "description": "Specifies the ephemeral disk placement for operating system disk. This property can be used by user in the request to choose the location i.e, cache disk or resource disk space for Ephemeral OS disk provisioning. By default For more information on Ephemeral OS disk size requirements, please refer : https://docs.microsoft.com/en-us/azure/virtual-machines/windows/ephemeral-os-disks#size-requirements",
```

Comment:
add the link for linux VMs too, https://docs.microsoft.com/en-us/azure/virtual-machines/linux/ephemeral-os-disks#size-requirements